lib/mongoid/persistable/updatable.rb

```diff
@@ -171,10 +172,10 @@ def update_document(options = {})
       # cleared in the before_update callback.
       #
       # @param [ Hash ] options The options.
-      def process_touch_option(options)
+      def process_touch_option(options, children)
```

Comment:
`children` needs to be added to docstring?

Reply:
Could you add at least the `touch` option also to the docstring?

Reply:
done